src/openrct2/ride/Vehicle.cpp

```diff
@@ -3431,8 +3431,7 @@ void Vehicle::FinishDeparting()
         audio_play_sound_at_location(SoundId::RideLaunch1, { x, y, z });
     }
 
-    if (curRide->mode != RideMode::Race && curRide->mode != RideMode::ContinuousCircuitBlockSectioned
-        && curRide->mode != RideMode::PoweredLaunchBlockSectioned)
+    if (curRide->mode != RideMode::Race && !(curRide->IsBlockSectioned()))
```

Comment:
```suggestion
    if (curRide->mode != RideMode::Race && !curRide->IsBlockSectioned())
```
Remove these parentheses as well.

One tip, when you fix the issue we open, make sure you set the conversation as resolved.

Reply:
Okay, will do, thanks